### PR TITLE
S04 us146 stats

### PR DESCRIPTION
**Pull request by cyrius83600**

### CODE REVIEW

File: src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.scss [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    }
+    
+    .survie {
+        grid-column: 2;
+        grid-row: 5;
+    }
+
+    // .modifyScore {
+    //     margin-top:6%;
+    // }
```
Comment (Kileana27) [inline src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.scss:185]: Dead code

File: src/app/service/api-data.service.ts [hunk 2]
```diff
@@ -13,6 +14,11 @@ export class ApiDataService {
 
   constructor() {}
 
+  getStatistiques(): Observable<Statistiques[]> {
```
Comment (Kileana27) [inline src/app/service/api-data.service.ts:17]: getStatistics in english

**Review by Kileana27 — CHANGES_REQUESTED** [review]
Just little changes :)

**Review by Kileana27 — APPROVED** [review]

File: src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<div class="modifying">
+    <p>Lancer vos dés 20 pour obtenir le niveau de chacune de vos statistiques </p>
+    <h1 class="modifyTitle">Explicatif des statistiques par mots clés</h1>
+
+      <div class="flexRow">
```
Comment (Benja-san) [inline src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html:5]: ul and li when you have a list

File: src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <div class="addiction">
+        <h2>ADDICTION</h2>
+        <div>
```
Comment (Benja-san) [inline src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html:8]: ul and li

File: src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        <h2>SURVIE</h2>
+        <p>Résistance environnement hostile</p>
+        <p>Capacité de survité</p>
+        <p>Prise de repères</p>
+        <p>Anticipation</p>
+      </div>
+    </div>
+
+    <h2 class="modifyScore">Modificateur du score</h2>
+    <p>On modifie le score en fonction de sa valeur</p>
```
Comment (Benja-san) [inline src/app/pages/explain-state-page-desktop/explain-state-page-desktop.component.html:83]: use only one p wrapping all your lines and use "span" to wrapp the lines behind

File: src/app/pages/stats-page/stats-page.component.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<div class="background-image">
+    <div class = "content-page">
+
+      <div class="selectStats">
+        <h1 class="title">Statistiques </h1>
+        <h2>Répartissez vos {{points}} points </h2>
+
+        @for(stat of statistiques; track stat)
```
Comment (Benja-san) [inline src/app/pages/stats-page/stats-page.component.html:8]: wrap this for in ul

File: src/app/pages/stats-page/stats-page.component.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
+<div class="background-image">
```
Comment (Benja-san) [inline src/app/pages/stats-page/stats-page.component.html:1]: section

File: src/app/pages/stats-page/stats-page.component.ts [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+  {
+    this.points = 5;
+    this.apiService.getStatistics().subscribe(
+      statistiques => {
+        this.statistiques = statistiques
+        for(let i = 0; i < 10; i++)
+          {
+            const max = 20;
+            let score = Math.round(Math.random()*max);
+            if(score >= 18) score = Math.round(score/2);
```
Comment (Benja-san) [inline src/app/pages/stats-page/stats-page.component.ts:81]: why ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**